dcrtimed/dcrtimed.go

```diff
@@ -187,6 +195,23 @@ func convertDigests(d []string) ([][sha256.Size]byte, error) {
 	return result, nil
 }
 
+func (d *DcrtimeStore) proxyWalletBalance(w http.ResponseWriter, r *http.Request) {
+	r.Body.Close()
```

Comment:
Shouldn't this be a defer body.Close somewhere?

Reply:
handled by closeBody now